tutorwebhookreceiver/plugin.py

```diff
@@ -14,7 +15,8 @@
         "EDX_OAUTH2_SECRET": "{{ 32|random_string }}",
     },
     "defaults": {
-        "DOCKER_IMAGE": "{{ DOCKER_REGISTRY }}webhookreceiver/webhookreceiver:latest",  # noqa: E501
+        "VERSION": __version__,
+        "DOCKER_IMAGE": "{{ DOCKER_REGISTRY }}webhookreceiver/webhookreceiver:{{ WEBHOOKRECEIVER_VERSION }}",  # noqa: E501
```

Comment:
Come to think of it: do we really want `{{ DOCKER_REGISTRY }}webhookreceiver/webhookreceiver:<version>`, as opposed to just `{{ DOCKER_REGISTRY }}webhookreceiver:<version>`? For the time being, the assumption is that you'll build this image yourself, as opposed to it being pushed to a public registry. And even if we ever push it to a public registry, it'll probably use a different prefix than `webhookreceiver`. What do you think?

Reply:
Yes that makes sense actually. I'll fix it